Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +11,48 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
+            try
+            {
+                byte[] byteMessage = Encoding.GetEncoding("iso-8859-1").GetBytes(message);
+                for (var i = 0; i < byteMessage.Length; i++)
+                {
+                    
+                    byteMessage[i] = (byte)(byteMessage[i] + shift);
+                    
+                }
+                message = System.Text.Encoding.GetEncoding("iso-8859-1").GetString(byteMessage);
+            }
+            catch(ArgumentNullException e)
+            {
+                return null;
+            }
+            
+
+           
+            
             return message;
         }
 
         public static string Decrypt(string message, byte shift)
```

Comment:
Decrypt can reuse encrypt (not quite, but almost). They both work exactly the same- only the shift is different.